src/chai/ManagedArray.hpp

```diff
@@ -107,7 +115,16 @@ class ManagedArray : public CHAICopyable
    * \param elems Number of elements in the array.
    * \param space Execution space in which to allocate the array.
    */
-  CHAI_HOST_DEVICE ManagedArray(size_t elems, ExecutionSpace space = NONE);
+  CHAI_HOST_DEVICE ManagedArray(
+      size_t elems,
+      ExecutionSpace space = NONE);
+
+  template<int N>
+  CHAI_HOST_DEVICE ManagedArray(
+      size_t elems,
+      std::array<chai::ExecutionSpace, N> spaces,
```

Comment:
@trws - this is failing to deduce `N` when I try and construct an array with a double brace initializer list. Any ideas?